RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
  0%|          | 0/100 [00:00<?, ?it/s]  1%|          | 1/100 [03:01<4:59:47, 181.70s/it]  2%|▏         | 2/100 [04:35<3:32:29, 130.10s/it] 49%|████▉     | 49/100 [04:38<02:46,  3.27s/it]   91%|█████████ | 91/100 [04:49<00:14,  1.61s/it] 94%|█████████▍| 94/100 [04:53<00:09,  1.59s/it]100%|██████████| 100/100 [04:53<00:00,  2.93s/it]
Exception ignored in: <function Pool.__del__ at 0x2b65eb545160>
Traceback (most recent call last):
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/multiprocessing/pool.py", line 268, in __del__
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/multiprocessing/queues.py", line 362, in put
AttributeError: 'NoneType' object has no attribute 'dumps'
